context.go

```diff
@@ -702,6 +704,9 @@ type FileHandler func(*parser.File)
 // which the future output will depend is returned.  This list will include both
 // Blueprints file paths as well as directory paths for cases where wildcard
 // subdirs are found.
+//
+// visitor will be called asynchronously, and will only be called once visitor for each
+// ancestor directory has completed.
```

Comment:
WalkBlueprintsFiles will not return until all calls to visitor have returned.

Reply:
done